vault/testing.go

```diff
@@ -926,6 +910,7 @@ func (c *TestCluster) Cleanup() {
 		go func() {
 			defer wg.Done()
 			if err := lc.stop(); err != nil {
+				// TODO do we care about this?
```

Comment:
We might not care about it, but I'd lean towards leaving it there, coz we never know when we begin to care :-)

Reply:
What I meant was, do we care about not seeing this error any more (now that we're stopping all output)?